assets/js/admin/course-edit.js

```diff
@@ -83,3 +84,19 @@ registerPlugin( 'sensei-course-video-progression-plugin', {
 	render: CourseVideoSidebar,
 	icon: null,
 } );
+
+/**
+ * Filters the course pricing sidebar toggle.
+ *
+ * @since 4.1.0
+ *
+ * @hook  senseiCoursePricingHide
+ * @param {boolean} hideCoursePricing
+ * @return {boolean}
+ */
+if ( ! applyFilters( 'senseiCoursePricingHide', false ) ) {
```

Comment:
Could we change the order so that it shows in the same order as in WCPC (i.e. Pricing, Access Period, Learning Mode, Video)?

Reply:
Jup set it on first place: [here](https://github.com/Automattic/sensei/pull/4810/files#diff-46b5b32718313cd62a309f3e5bb5b44af7fd7d418aa97369ac2e33b8204b7d35R81)